<launch>
    <!-- Start the odometry transformer node -->
    <node name="odometry_transformer_node" pkg="unity_ros_tf" type="odometry_transformer2.py" output="screen">
        <param name="parent_frame" value="multijackal_02/odom" /> <!--map-->>
        <param name="child_frame" value="multijackal_02/base_footprint" />        
        <param name="position_offset" value="[0.0, 0.0, 0.0] " />
        <param name="euler_rotation_offset" value="[0.0, 0.0, 0.0]" />
        <remap from="/original_odom" to="/multijackal_02/odom"/>
        <remap from="/global_odom" to="/global/multijackal_02/odom"/>
        <remap from="/is_climbing" to="/is_climbing"/>
    </node>

    <!-- Start the occupancy grid transformer node
    <node name="occupancy_grid_transformer_node" pkg="unity_ros_tf" type="occupancy_grid_transformer.py" output="screen">
        <param name="parent_frame" value="parent_frame" />
        <param name="child_frame" value="child_frame" />
        <param name="map_origin_offset" value="[0.0, 0.0, 0.0]" />
        <param name="odom_topic" value="/odom" />
        <param name="trigger_topic" value="/trigger" />
        <param name="input_map_topic" value="/input_map" />
        <param name="output_map_topic" value="/output_map" />
    </node> -->
</launch>